Adding Send Signal from Client in Python (#1631)
temporalio/documentation: docs-src/python/how-to-send-a-signal-from-a-client-in-python.md

---
id: how-to-send-a-signal-from-a-client-in-python
title: How to send a Signal from a Client in Python
sidebar_label: Send a Signal from a Client

> description: Send a Signal from a Client

Reviewer: To send a Signal to a Workflow Execution from Client code, use the `signal()` method on the Workflow handle.

tags:
  - developer-guide
  - sdk
  - python
---

> To send a Signal to a Workflow Execution from Client code, use the [`signal()`](https://python.temporal.io/temporalio.workflow.html#signal) API in the Workflow interface.

Reviewer 2: ```suggestion
To send a Signal to a Workflow Execution from Client code, use the [`signal()`](https://python.temporal.io/temporalio.workflow.html#signal) method on the Workflow handle.
```

Also, this assumes that the user at this point knows what a "handle" is. If not, may need to clarify.

> To send a Signal to a Workflow Execution from a Client, use the [`signal()`](https://python.temporal.io/temporalio.workflow.html#signal) API in the Workflow interface, from the Client code.

Reviewer: Does this simplification retain the same sense?
```suggestion
To send a Signal to a Workflow Execution from Client code, use the [`signal()`](https://python.temporal.io/temporalio.workflow.html#signal) API in the Workflow interface.
```
